pkgs/development/libraries/hunspell/dictionaries.nix

```diff
@@ -821,4 +823,27 @@ in rec {
     readmeFile = "README_he_IL.txt";
     license = with lib.licenses; [ agpl3Plus ];
   };
+
+  /* THAI */
+
+  th_TH = th-th;
+  th-th = mkDict {
+    pname = "hunspell-dict-th-th";
+    version = "experimental-2021-11-17";
+    dictFileName = "th_TH";
+    readmeFile = "README.md";
+    src = fetchFromGitHub {
+      owner = "SyafiqHadzir";
+      repo = "Hunspell-TH";
+      rev = "f119e58e5f6954965d6abd683e7d4c4b9be2684f";
+      sha256 = "sha256-hwqKvuLxbtzxfyQ5YhC/sdb3QQwxCfzgDOHeatxDjxM=";
+    };
+    meta = with lib; {
+      description = "Hunspell dictionary for Central Thai (Thailand)";
+      homepage = "https://hunspell-th.syafiqhadzir.dev";
```

Comment:
are you sure this url is ok? I get cloudflare Invalid SSL certificate error when trying to visit it.

Reply:
Yeah. I emailed the project maintainer about it already. I did not get a response on this. Maybe he doesn't check his email. It would be this or the project page.

Reply:
Maybe we can use github repo instead. I'm bit hesitant to include this since we link to these urls from many places like search.nixos.org I think we should keep number of broken or suspicious links to minimum.

Reply:
I think I agree. The author got back and said it _might_ be up in January, but it's unreasonable to ask that of someone and probably isn't that relevant versus the straight source. Updated.